pom.xml

```diff
@@ -10,9 +10,10 @@
   <artifactId>stash-pullrequest-builder</artifactId>
   <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
+  
   <name>Stash Pullrequest Builder Plugin</name>
-  <description>This Jenkins plugin builds pull requests from Stash and will report the test results.</description>
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/Stash+pullrequest+builder+plugin</url>
+  <description>This Jenkins plugin builds pull requests from Stash and will report the test results</description>
```

Comment:
`will report` -> `reports`
It's already clear that reporting happens after building; future tense is not justified.
Actually, how about this?
```
This plugin builds Stash pull requests and posts the build results on Stash
```
Just look at other plugin descriptions. "Jenkins" is implied.